jvcl/run/JvAppIniStorage.pas

```diff
@@ -704,8 +706,21 @@ procedure TJvCustomAppIniStorage.DeleteSubTreeInt(const Path: string);
           IniFile.EraseSection(Sections[I])
       else
         for I := 0 to Sections.Count - 1 do
-          if Pos(TopSection, Sections[I]) = 1 then
```

Comment:
Why not just change this to

```
if Pos(TopSection + '\', Sections[I] + '\') = 1 then
```

Reply:
I think that's much cleaner and doesn't clutter the code that much and should also be faster since we wouldn't need ``Copy()``, and 1 or 2 ``SameText()``